src/themes/amsterdam/global_styling/variables/_typography.ts

```diff
@@ -34,6 +34,7 @@ export const fontBase: _EuiThemeFontBase = {
 
   // Careful using ligatures. Code editors like ACE will often error because of width calculations
   featureSettings: "'calt' 1, 'kern' 1, 'liga' 1",
+  fontSizeUnit: 'rem',
```

Comment:
[naming things is hard] now that I realize `line-height` is using this as well as `font-size`, I'm wondering if this var name still makes sense. Maybe just

```suggestion
  sizeUnit: 'rem',
```

?

Reply:
I lean toward `defaultUnits` myself. When I see the object `font.defaultUnits` I immediately think "pixesl, ems, or rems."

Reply:
I'm leaning towards `defaultUnits` as well! Going to make that change here 🎉

Reply:
Token rename: https://github.com/elastic/eui/pull/7133/commits/c94844c56ae180f8c43938d0c3ff66ecde1a2eda

Far more opinionated rename of all `measurement`-adjacent to `unit` instead: https://github.com/elastic/eui/pull/7133/commits/7d3c34a85e6e6e04ac5aab0191444749e2e6d317

I'd be curious to hear what you think about the 2nd commit - for me personally, as a dev with over a decade of CSS experience, `measurement` means very little to me, whereas `unit` has a very specific connotation within CSS font sizes and I feel it more accurately describes the enum. That being said, I'm being pretty opinionated with that change, so if you're not a fan of it or are worried about downstream implications, I'm definitely open to reverting it!

Reply:
I'm looking at the changes now and I think you're onto something with this new naming convention.

To my thinking
* `16` is a **measurement**
* `px` are the **units**

Units mean a lot more to me too. The functions are clearer with the new names.

Reply:
Great call on the distinction. It sounds like you're good with moving ahead with the total rename - if so, I'll merge the PR once you've approved. Thanks again Trevor!